openedx/features/edly/tests/factories.py

```diff
@@ -51,3 +53,43 @@ class Meta(object):
     edx_organization = SubFactory(OrganizationFactory)
     lms_site = SubFactory(SiteFactory)
     studio_site = SubFactory(SiteFactory)
+
+
+class EdlyUserProfileFactory(DjangoModelFactory):
+    """
+    Factory class for "EdlyUserProfile" model.
+    """
+    class Meta(object):
+        model = EdlyUserProfile
+        django_get_or_create = ('user', )
+
+    id = Sequence(u'{0}'.format)
```

Comment:
we should not explicitly specify `id`. remove this line.

Reply:
Done